testsuite/features/build_validation/migration/min_salt_migration.feature

```diff
@@ -2,60 +2,37 @@
 # Licensed under the terms of the MIT license.
 
 # This feature depends on
-# - srv_sync_channels.feature which syncs SLES 15 SP4 and the client tools.
+# - srv_sync_all_products.feature which syncs SLES 15 SP5 and the client tools.
 # We also test 'Bootstrapping using the command line' in this feature
 
 @scope_salt
 @scope_salt_ssh
-@scope_virtualization
-@virthost_kvm
-Feature: Migrate Salt to bundled Salt on a nested Minion VM
+@sle15sp5_minion
```

Comment:
The card describes that we need a dedicated new VM for this minion, we can't use `sle15sp5_minion`